pilot/pkg/networking/core/v1alpha3/httproute_test.go

```diff
@@ -781,16 +863,16 @@ func TestSidecarOutboundHTTPRouteConfig(t *testing.T) {
 				"bookinfo.com:70":        {"bookinfo.com:70": true, "*.bookinfo.com:70": true},
 				"test-headless.com:8888": {"test-headless.com:8888": true, "*.test-headless.com:8888": true},
 				"test-private-2.com:60": {
-					"test-private-2.com": true, "test-private-2.com:60": true, "9.9.9.10": true, "9.9.9.10:60": true,
+					"test-private-2.com:60": true, "9.9.9.10:60": true,
 				},
 				"test-private.com:70": {
-					"test-private.com": true, "test-private.com:70": true, "9.9.9.9": true, "9.9.9.9:70": true,
+					"test-private.com:70": true, "9.9.9.9:70": true,
 				},
 				"test-private.com:80": {
 					"test-private.com": true, "test-private.com:80": true, "9.9.9.9": true, "9.9.9.9:80": true,
 				},
 				"test.com:8080": {
-					"test.com:8080": true, "test.com": true, "8.8.8.8": true, "8.8.8.8:8080": true,
+					"test.com:8080": true, "8.8.8.8:8080": true,
```

Comment:
I am confused by this change. Won't this cause a regression since now the port must be 8080?

Reply:
No., this has been the case always. It is just that the test case is not validating what domains should not be there - just validating what should be there earlier. I added that check so had to remove this

Reply:
Just to be sure, I have added the same validation in master and the test failed with this
```
expected vhost domain test-private-2.com in vhost test-private-2.com:60, for route 7443 not found. got domains [test-private-2.com:60 9.9.9.10:60]
```